src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -42,6 +46,38 @@ public class DurableOrchestrationClient : DurableOrchestrationClientBase
             this.attribute = attribute;
         }
 
+        /// <summary>
+        /// Constructor for <see cref="DurableOrchestrationClient"/>
+        /// </summary>
+        /// <param name="connectionName">Azure Storage connection name.</param>
+        /// <param name="taskHubName">Name of task hub.</param>
+        public DurableOrchestrationClient(string connectionName, string taskHubName)
+        {
+            this.config = new DurableTaskExtension();
+            JobHostConfiguration jobHostConfiguration =
+                new JobHostConfiguration(connectionName)
+                {
+                    LoggerFactory = new LoggerFactory(),
+                };
+            ExtensionConfigContext extensionConfigContext = new ExtensionConfigContext
+            {
+                Config = new JobHostConfiguration(connectionName),
+            };
+            ((IExtensionConfigProvider)this.config).Initialize(extensionConfigContext);
```

Comment:
I worry about using types like `ExtensionConfigContext` in our extension code because they're really only "supposed" to be used by the WebJobs host. My fear is that some behavior could be changed by the functions team in a way we don't expect and break customers who depend on this.

Reply:
Yes, I agree with you. When I was calling this method the feeling was that we are hacking the API-s and trying to find workarounds.

Thank you!